app/jsx/components/Search2Comp.jsx

```diff
@@ -1,27 +1,79 @@
-// ##### Search Component ##### //
+// ##### Search Component - With Search Controls ##### //
 
 import React from 'react'
 import Form from 'react-router-form'
 
+class CustomRadioInput extends React.Component {
+  render() {
+    return (
+      <input type="radio" id="c-search2__refine-campus" name={this.props.name} value={this.props.value} onClick={this.props.handleRadioSelect}/>
+  )}
+}
+
+class SearchControls extends React.Component {
+  constructor(props) {
+    super(props)
+    this.handleRadioSelect = this.props.handleRadioSelect.bind(this)
```

Comment:
Unneeded, see below... In fact, the whole constructor for SearchControls isn't needed.